lib/weapon.rb

```diff
@@ -0,0 +1,24 @@
+class Weapon
+  attr_reader :choice
+  CHART = {
+    rock: [:scissors, :lizard],
+    paper: [:rock, :spock],
+    scissors: [:paper, :lizard],
+    lizard: [:paper, :spock],
+    spock: [:rock, :scissors]
+  }
+
```

Comment:
Nice use of a default param

Reply:
The only drawback is that it's not entirely clear, in `app.rb` what's going on here.  If you had to pass a param to indicate that a random choice was required, that would be clearer but less concise.  So there's a tradeoff.